File: WebApplication/src/nomenclatures/dtos/settlements/settlement.dto.ts
import { NomenclatureDto } from "../nomenclature.dto";
import { DistrictDto } from "./district.dto";
import { MunicipalityDto } from "./municipality.dto";

export class SettlementDto extends NomenclatureDto {
    districtId: number;
    district: DistrictDto;
    municipalityId: number;
    municipality: MunicipalityDto;
}

File: Sc.Models/Dtos/Nomenclatures/Settlements/SettlementDto.cs
using Sc.Models.Dtos.Base.Nomenclatures;

namespace Sc.Models.Dtos.Nomenclatures.Settlements
{
    public class SettlementDto : NomenclatureDto
    {
        public int DistrictId { get; set; }
        public DistrictDto District { get; set; }
        public int MunicipalityId { get; set; }
        public MunicipalityDto Municipality { get; set; }
    }
}
